openedx/core/djangoapps/user_api/preferences/tests/test_views.py

```diff
@@ -137,10 +137,14 @@ def _do_create_preferences_test(self, is_active):
         if not is_active:
             self.user.is_active = False
             self.user.save()
-        self.send_patch(self.client, {
-            "dict_pref": {"int_key": 10},
-            "string_pref": "value",
-        })
+        self.send_patch(
+            self.client,
+            {
+                "dict_pref": {"int_key": 10},
+                "string_pref": "value",
+            },
+            expected_status=204
```

Comment:
These are for the user preferences API, which we are not updating at this time.  This PR is for the user accounts API.